src/Slices/CompileDetails/UI/CompileStageReportTable.tsx

```diff
@@ -37,7 +38,11 @@ export const CompileStageReportTable: React.FC<Props> = ({
     <Table {...props} variant={TableVariant.compact}>
       <Thead>
         <Tr>
-          <Th aria-hidden key="toggle" />
+          <Th
+            aria-hidden
```

Comment:
just found out that the aria-hidden isn't needed when adding the screenReaderText, no need to make the changes, just an FIY